lisa/operating_system.py

```diff
@@ -179,7 +201,23 @@ def install_packages(
         if self._first_time_installation:
             self._first_time_installation = False
             self._initialize_package_installation()
-        self._install_packages(package_names)
+
+        return self._install_packages(package_names, signed)
+
+    # TODO: Immplement update_packages
+    def update_packages(
+        self, packages: Union[str, Tool, Type[Tool], List[Union[str, Tool, Type[Tool]]]]
+    ) -> None:
+        pass
+
+    # TODO: Immplement query_packages
+    def query_packages(
+        self, packages: Union[str, Tool, Type[Tool], List[Union[str, Tool, Type[Tool]]]]
+    ) -> List[str]:
+        pass
+
+    def get_os_version(self) -> OsVersion:
```

Comment:
Because there is no common logic, it doesn't need a wrapper here.

Reply:
> Because there is no common logic, it doesn't need a wrapper here.

Done